examples/plot_6_function_identifier.py

```diff
@@ -34,7 +39,8 @@
 # Also note that global variables of the script have intersphinx references
 # added to them automatically (e.g., ``fig`` and ``fig.text`` below).
```

Comment:
This isn't from this PR but since we are cleaning up the example, I think this text block may be better suited at the bottom of the first text block (~line 7). Could we move this up?

In this text block could we explain that we are demonstrating this functionality by using the internal function `identify_names` to obtain the full resolved import path for all backreferences in the code block above (would be nice to add reference``` :ref:`references_to_examples` ```), and just explicitly say that the left side are functions as written in the code block and the right is the full resolved import path for that function (though I know it's fairly obvious already).

Reply:
Thanks for the comment, please see c7cf0116e427acff2f4b01d26b396e251ea17278. Hope I got it right, feel free to suggest any changes.